test/integration/pre.ts

```diff
@@ -37,7 +38,13 @@ import shell from 'shelljs';
   }
 
   try {
-    await nginx.start();
+    const staticBuildDirectory = path.join(__dirname, '..', '..', 'test', 'shared', 'sampleApplication', 'out');
```

Comment:
We are already in the `test` directory, so this can be simplified.

```suggestion
    const staticBuildDirectory = path.join(__dirname, '..', 'shared', 'sampleApplication', 'out');
```